projects/lib/src/lib/services/luigi-nodes/children-nodes.service.ts

```diff
@@ -63,21 +63,31 @@ export class ChildrenNodesService {
     );
 
     childrenNodes.forEach((child) => {
-      child.context = { ...child.context, entityContext };
+      child.context = { ...(child.context ?? ({} as any)), entityContext };
       child.onNodeActivation =
         this.nodeUtilsService.retrieveGlobalHelpContext();
     });
 
     const nodes = await Promise.all(
       childrenNodes
         .filter((child) => visibleForContext(child.context, child))
```

Comment:
I think we can change in the model tha that node.context is always there (it's set for all the nodes)
https://github.com/openmfp/portal-server-lib/blob/c25ad525a56dee4158018a198cd2f1139cacb169/src/config/luigi/luigi-data/node-extended-data.service.ts#L22
then we are better with all of the changes, no need to check that much , and the reson it worked so far without the checks